plugins/outputs/nats/README.md

```diff
@@ -69,9 +107,9 @@ to use them.
   ## Jetstream specific configuration. If not nil, it will assume Jetstream context.
   ## Since this is a table, it should be present at the end of the plugin section. Else you can use inline table format.
   # [outputs.nats.jetstream]
-    ## Name of the stream, required when using jetstream. Telegraf will
-    ## use the union of the above subject and below the subjects array.
+    ## Name of the stream, required when using jetstream.
     # name = ""
+    ## List of subjects to register on the stream
```

Comment:
Aren't those the "streams" created, updated or used? If so, could we please add this as a note...